app/ide-desktop/lib/dashboard/src/components/dashboard/ProjectIcon.tsx

```diff
@@ -103,13 +103,12 @@ export default function ProjectIcon(props: ProjectIconProps) {
           type: newState,
         })
         if (!backendModule.IS_OPENING_OR_OPENED[newState]) {
-          // eslint-disable-next-line @typescript-eslint/naming-convention, @typescript-eslint/no-unused-vars
-          const { opened_by, ...newProjectState2 } = newProjectState
+          // eslint-disable-next-line @typescript-eslint/no-unused-vars
+          const { openedBy, ...newProjectState2 } = newProjectState
```

Comment:
i guess we should change this to use the `object.omit`, so that it no longer needs the lint disable: <https://github.com/enso-org/enso/blob/fc557f8fd2804379e4dad52b44d484b5aa36b748/app/ide-desktop/lib/dashboard/src/utilities/object.ts#L95>

... on that note it seems like there's a typo there. oh well... i might fix that at some point in the future

Reply:
no the typo is in the docs, which say "runtie array" :sweat_smile: 

`Ks` is short for keys, maybe it should be renamed too though

Reply:
Typo fixed :)